docs/index.md

```diff
@@ -67,7 +67,7 @@ Example applications are also provided in Python. For example,
     ./bin/spark-submit examples/src/main/python/pi.py 10
 
 Spark also provides an experimental [R API](sparkr.html) since 1.4 (only DataFrames APIs included).
```

Comment:
tbh, I'm not sure this should be called "an experimental [R API]"

Reply:
Should we remove the "experimental" adjective?

Reply:
let's remove `experimental`, WDYT?

Reply:
OK, `experimental` has been removed. https://github.com/apache/spark/pull/23145/files#diff-1a523bd9fa0dbf998008b37579210e12